cmd/ecslog/main_test.go

```diff
@@ -96,9 +96,16 @@ var mainTestCases = []mainTestCase{
 		regexp.MustCompile(`^\n    foo: "bar"\n    spam: "eggs"\n$`),
 		nil,
 	},
+	{
+		"ecslog --include-fields ...all the title fields...",
+		[]string{"ecslog", "-i", "foo", "./testdata/exclude-fields.log"},
+		0,
+		regexp.MustCompile(`^\[2021-01-19T22:51:12.142Z\]  INFO: hi\n    foo: "bar"\n    spam: \n$`),
```

Comment:
Shouldn't the `spam: ...` line be complete gone here?

Reply:
yes it, fixed